crates/engine/config/src/trusted_documents.rs

```diff
@@ -0,0 +1,33 @@
+pub use gateway_config::LogLevel;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct TrustedDocumentsConfig {
+    pub unknown_trusted_document_id_log_level: LogLevel,
+    pub document_id_and_query_mismatch_log_level: LogLevel,
+    pub unknown_inline_document_log_level: LogLevel,
+}
+
+impl Default for TrustedDocumentsConfig {
+    fn default() -> Self {
+        gateway_config::TrustedDocumentsConfig::default().into()
+    }
+}
+
+impl From<gateway_config::TrustedDocumentsConfig> for TrustedDocumentsConfig {
+    fn from(
+        gateway_config::TrustedDocumentsConfig {
+            unknown_trusted_document_id_log_level,
```

Comment:
perhaps `unknown_id_log_level` to be shorter?

Reply:
done